### PR TITLE
docs: moved local docker setup into local setup file

### PR DESCRIPTION
**Pull request by cmccormack**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [X] My changes do not use shortened URLs or affiliate links.

This PR was requested by @raisedadead and other FCC devs to remove redundant setup files to prevent having to make the same changes in multiple places.  @freeCodeCamp/dev-team  please review and provide suggestions to help improve the document if needed.

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -105,36 +103,52 @@ You can skip running freeCodeCamp locally if you are simply editing files, perfo
...
-| --------------------------------------------------------------------------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+Software required for both Docker and Local builds:
+
+| Prerequisite | Version | Notes |
+| ------------ | ------- | ----- |
+| [Node.js](http://nodejs.org)| `10.x`  | [LTS Schedule](https://github.com/nodejs/Release#release-schedule) |
+| npm (comes bundled with Node)| `6.x`   | Does not have LTS releases, we use the version bundled with Node LTS |
+
+**Docker Build additional prerequisite:**
+| Prerequisite | Version | Notes |
```
Comment (raisedadead) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion

| Prerequisite | Version | Notes |
```

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -105,36 +103,52 @@ You can skip running freeCodeCamp locally if you are simply editing files, perfo
...
+| ------------ | ------- | ----- |
+| [Docker CE](https://docs.docker.com/install/) | `Stable` |       |
```
Comment (raisedadead) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion
| [Docker CE](https://docs.docker.com/install/) | `Stable` | - |
```

File: docs/how-to-setup-freecodecamp-locally.md [hunk 3]
```diff
@@ -105,36 +103,52 @@ You can skip running freeCodeCamp locally if you are simply editing files, perfo
...
+
+**Local Build additional prerequisite:**
+| Prerequisite | Version | Notes |
```
Comment (raisedadead) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion

| Prerequisite | Version | Notes |
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
image


You can use the preview button to check if the table will look good on commit.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 4]
```diff
@@ -105,36 +103,54 @@ You can skip running freeCodeCamp locally if you are simply editing files, perfo
 
 [Skip running freeCodeCamp locally](#making-changes-to-your-clone-of-freecodecamp-locally)
 
+There are currently two methods to run freeCodeCamp locally:
+- Docker (recommended)
+- Local
+
+Only **one** of these methods is necessary. If you are having issues using one method, try using the other. If you do run into issues with either method, first perform a web search for your issue and see if it has already been answered. If you cannot find a solution, please search our GitHub [issues](https://github.com/freeCodeCamp/freeCodeCamp/issues) page for a solution and report the issue if it has not yet been reported.
```
Comment (raisedadead) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion
You are required to follow any **one** of the above methods. 

The Docker setup will ideally result in fewer errors during the installation process and aims to have the best developer experience. We use Docker to install and run the additional software dependencies behind the scenes. This should make it hassle free and consistent experience across most device/OS types.

If you are having issues using one method, try using the other. If you do run into issues with either method, first perform a web search for your issue and see if it has already been answered. If you cannot find a solution, please search our GitHub [issues](https://github.com/freeCodeCamp/freeCodeCamp/issues) page for a solution and report the issue if it has not yet been reported.

And as always, feel free to hop on to our [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors), for quick queries.
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
I moved around some things to explain the context better. Let me know what you think. Thanks this is good to go.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 5]
```diff
@@ -1,11 +1,10 @@
 # Set up freeCodeCamp locally
-
-As of 8 March 2019, please consider helping us test our new guide on how to [setup freeCodeCamp locally using Docker](/docs/how-to-setup-freecodecamp-locally-using-docker.md) instead of using this guide. It *should* result in fewer, if not zero, errors but we won't know until enough devs try it.
-
 Follow these guidelines for getting freeCodeCamp locally on your system. This is highly recommended if you want to contribute regularly.
 
 Some of the contribution workflows, like previewing pages for the guide or the coding challenges, debugging and fixing bugs in codebase, requires you to have freeCodeCamp running locally.
 
+The new Docker setup procedure *should* result in fewer, if not zero, errors during the setup process and make it easier for you. We use Docker to install and run many of the software dependencies which means that you don't have to install services like MongoDB or MailHog on your computer or worry about ensuring you are using the correct version.
```
[the remark this answers is not in the document]
Comment (raisedadead) [reply docs/how-to-setup-freecodecamp-locally.md]: Remove here, move until the users reach the actual setup.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 6]
```diff
@@ -105,36 +103,54 @@ You can skip running freeCodeCamp locally if you are simply editing files, perfo
...
+| ------------ | ------- | ----- |
+| [Node.js](http://nodejs.org)| `10.x`  | [LTS Schedule](https://github.com/nodejs/Release#release-schedule) |
+| npm (comes bundled with Node)| `6.x`   | Does not have LTS releases, we use the version bundled with Node LTS |
+
+**Docker Build additional prerequisite:**
+
+| Prerequisite | Version | Notes |
+| ------------ | ------- | ----- |
+| [Docker CE](https://docs.docker.com/install/) | `Stable` | - |
+
```
[the remark this answers is not in the document]
Comment (cmccormack) [reply docs/how-to-setup-freecodecamp-locally.md]: Thank you - glad you added it to this PR rather than to a new PR against the Docker setup doc!

**Review by raisedadead — APPROVED** [review]

File: docs/how-to-setup-freecodecamp-locally.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply docs/how-to-setup-freecodecamp-locally.md]: No problem!  I'm glad I checked first...

**Pull request merged**